src/EditorFeatures/TestUtilities/Traits.cs

```diff
@@ -96,6 +96,7 @@ public static class Features
             public const string Formatting = "Formatting";
             public const string GoToDefinition = "GoToDefinition";
             public const string GoToImplementation = "GoToImplementation";
+            public const string GoToNextAndPreviousMember = "GoToNextAndPreviousMember";
```

Comment:
Maybe the feature name should be shortened to "Go to adjacent member"? All this "NextAndPrevious" stuff is wearing me out.